.editorconfig

```diff
@@ -0,0 +1,23 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```

Comment:
We have indent size = 3 for Perl code

Reply:
What is the first batch I should reindent?
Should I leave artistic indentation in place?
```perl
   my ($package, undef, $line) = caller 0;
   @_ = map { (my $temp = $_) =~ s/\n/\n# /g; $temp; }
        map { defined $_ ? $_ : 'undef' }
        @_;
```

Reply:
Yes, please, leave them.